setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+from piqe_utils import __version__
+
+setup(
+    name="piqe-utils",
+    version=__version__,
+    author="PIQE Libraries Team",
+    author_email="portfolio-integration-qe@redhat.com",
+    description="PIQE OpenShift Python Libraries.",
```

Comment:
May be change this description?

Reply:
Sure will do the change. Copy paste error :-)